streams/src/main/java/org/apache/kafka/streams/kstream/KStreamBuilder.java

```diff
@@ -40,10 +40,11 @@ public KStreamBuilder() {
     }
 
     /**
-     * Creates a {@link KStream} instance from the specified topics.
+     * Creates a {@link KStream} instance for the specified topics.
      * The default deserializers specified in the config are used.
      *
      * @param topics    the topic names; must contain at least one topic name
+     * @return a {@link KStream} for the specified topics
```

Comment:
When a stream is created from multiple topics, we do not have any ordering semantics across those topics, but within a single topic we still follow the within-partition ordering.